Hmm, I am actually still getting

```
    simulate.go:263: error on block  274/1000, operation (522/766) from x/authz:
        requested amount is more than spend limit: insufficient funds [cosmos/cosmos-sdk/types/errors/errors.go:273]
        Comment: requested amount is more than spend limit: insufficient funds
```

with this branch

--- a/simapp/helpers/test_helpers.go
+++ b/simapp/helpers/test_helpers.go
@@ -13,8 +13,10 @@ import (
 )
 
 // SimAppChainID hardcoded chainID for simulation
+// TODO: Implement a tool to check for gas growth. For now,
+// we must increase this across the board for longer runs to work
 const (
-	DefaultGenTxGas = 1000000
+	DefaultGenTxGas = 2000000
 	SimAppChainID   = "simulation-app"
 )
 

--- a/x/authz/simulation/operations.go
+++ b/x/authz/simulation/operations.go
@@ -249,7 +249,7 @@ func SimulateMsgExec(ak authz.AccountKeeper, bk authz.BankKeeper, k keeper.Keepe
 			return simtypes.NoOpMsg(authz.ModuleName, TypeMsgExec, "not a send authorization"), nil, nil
 		}
 
-		if sendAuth.SpendLimit.IsAllLTE(coins) && coins.DenomsSubsetOf(sendAuth.SpendLimit) {
+		if !sendAuth.SpendLimit.IsAllGTE(coins) || !coins.DenomsSubsetOf(sendAuth.SpendLimit) {
 			return simtypes.NoOpMsg(authz.ModuleName, TypeMsgExec, "over spend limit"), nil, nil
 		}
 

